marketing/tasks.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2020 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Worker tasks for marketing features
+"""
+
+# Imports #####################################################################
+
+import logging
+from datetime import timedelta
+
+from django.conf import settings
+from django.utils.timezone import now
+from django.db.models import Prefetch
+from huey import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from marketing.models import Subscriber, EmailTemplate, SentEmail
+from marketing.utils import render_and_dispatch_email
+
+
+# Logging #####################################################################
+
+logger = logging.getLogger(__name__)
+
+
+# Functions ###################################################################
+
+def get_eligible_subscribers_queryset(template: EmailTemplate):
+    """
+    Creates and returns a queryset for all eligible subscribers for
+    the given template.
+
+    For better understanding following is a timeline containing filter days
+                   +-------------------+
+    <start>--------|<gte>--1-day--<lte>|--------send-after-days----<now>--
+                   +-------------------+
+    Eligible subscribers are those which are created in our one day window
+
+    Uses prefetch to reduce DB queries.
+    """
+    gte = now() - timedelta(days=template.send_after_days + 1)
+    lte = now() - timedelta(days=template.send_after_days)
```

Comment:
I think it should be `lt` here to avoid overlaps.
```suggestion
    lt = now() - timedelta(days=template.send_after_days)
```